src/main/java/org/lwes/emitter/EmitterGroupFilter.java

```diff
@@ -39,4 +42,39 @@ public String toString() {
     for (String e : filtered) sb.append(e).append(" ");
     return sb.toString();
   }
+
+
+  // Helper method
+  public static EmitterGroupFilter fromProperties(Properties props, String prefix) {
```

Comment:
Is this exact copy/paste of getEmitterGroupFilter

Reply:
Yes, it moved from EmitterGroup for cleanliness.